storage/src/cache/worker.rs

```diff
@@ -289,15 +292,25 @@ impl AsyncWorkerMgr {
         // Allocate network bandwidth budget
         if let Some(limiter) = &self.prefetch_limiter {
             let size = match msg {
-                AsyncPrefetchMessage::BlobPrefetch(state, _blob_cache, _offset, size) => {
-                    if state.load(Ordering::Acquire) > 0 {
+                AsyncPrefetchMessage::BlobPrefetch(blob_cache, _offset, size) => {
+                    if blob_cache
```

Comment:
We could simplify code further by adding a helper function for the repeated code snippet to get state.

Reply:
Wrapped it into a helper function now